### ISSUE TITLE
Test batch size > 1?

### ISSUE THREAD

**Issue by burlinge** [junyanz/pytorch-CycleGAN-and-pix2pix#178]
Greetings,

I am stitching together 256 x 256 generated tiles to make large images and can imagine that batch rather than instance statistics at test/normalization time would benefit my final results.

If I'm not mistaken, the lua pix2pix test code allowed for variable test batch sizes. Will pytorch pix2pix test code support batchSize > 1 in the future?

Thanks and all the best,

Erik

**Comment by SsnL** [comment]
unless you train with batch norm, it won't make a difference.

**Comment by burlinge** [comment]
Thanks for the reply, @SsnL.

Yes, we are training using batch norm, i.e. --norm batch, so it could make a substantial difference.

**Comment by SsnL** [comment]
Cool. The reason nb is forced as 1 is that the image output code doesn't loop over the batch. You can pretty easily change that to support it :)

**Comment by burlinge** [comment]
Cheers.
